lib/beaker/options/presets.rb

```diff
@@ -13,7 +13,7 @@ class Presets
       # so that preferred and "fallback" values work as expected.
       ENVIRONMENT_SPEC = {
         :home                 => 'HOME',
-        :project              => ['BEAKER_PROJECT', 'BEAKER_project'],
+        :project              => ['BEAKER_PROJECT', 'BEAKER_project', 'JOB_NAME'],
```

Comment:
Is 'JOB_NAME' a requirement here?  Beaker env vars are of the form BEAKER_*** (unless they are historic and pending deprecation).